src/refract/RenderJSONVisitor.cc

```diff
@@ -223,6 +225,61 @@ namespace refract
         assign(renderer.get());
     }
 
+    namespace {
```

Comment:
Please move this namespace to be just below the `namespace refract` declaration. Also please include the other `getSample`, `getDefault`, `isTypeAttribute` functions in this namespace.